x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/common/constants/comparators.ts

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Should we remove `threshold` references in this file to keep translation keys generic? Ex: `xpack.triggersActionsUI.common.constants.comparators.isAboveLabel`.